cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -73,6 +81,43 @@ class CallSubqueryTest extends DocumentingTest {
       graphViz()
     }
 
+    section("Semantics, call-semantics") {
```

Comment:
Didn't you mean to have the comma outside the string?
```suggestion
    section("Semantics", "call-semantics") {
```

Reply:
Yes. Thank you.